src/dialogs/PaymentErrorDialog.vue

```diff
@@ -103,5 +103,21 @@ export default class PaymentErrorDialog extends Vue {
   get numWarnings (): number {
     return this.warnings?.length || 0
   }
+
+  /** The formated error message */
+  protected formatedError = []
+  protected mounted (): void {
+    this.formatedError = this.errors.map((item:any) => (
```

Comment:
The local variable and code in mounted() could be simply replaced with a local getter.

Reply:
This is code is removed.